packages/eas-cli/src/commands/update/index.ts

```diff
@@ -586,15 +608,32 @@ async function getRuntimeVersionObjectAsync(
     }
   }
 
-  return Object.fromEntries(
-    platforms.map(platform => [
-      platform,
-      nullthrows(
-        Updates.getRuntimeVersion(exp, platform),
-        `Unable to determine runtime version for ${platform}`
-      ),
-    ])
-  );
+  try {
+    return transformRuntimeVersions(exp, platforms);
+  } catch (error: any) {
+    ora().start().fail(error.message);
+
+    const runConfig = await selectAsync(`Do you want us to run 'eas update:configure' for you?`, [
+      { title: 'Yes', value: true },
+      {
+        title: 'No, I will edit files or run the command manually (EAS CLI exits)',
```

Comment:
```suggestion
        title: 'No, I will set the runtime version manually (EAS CLI exits)',
```

Reply:
How strong is you opinion on that phrasing? 

The `eas update:configure` besides setting the runtime version performs edit of other files, to prepare the project for EAS Update, that's why I have decided to went with that sentence.

Reply:
The 'I will edit files' part is a bit inconclusive, and you are only checking for runtime version, so it's not guaranteed that they need to run that command, they could have made any change that causes this. So if you're checking runtime version, I think the instructions should be as narrow as possible to set runtime version. Otherwise, instead of saying 'cannot detect runtime version' you should say 'it seems your project is not configured for EAS Update yet' (but I prefer to just stick to the runtime version.

Reply:
What do you think about: 
```
"No, I will set the runtime version or run the command manually (EAS CLI exits)"
```
since the question above refers only to that action?

And then, we can rephrase the question above: 
```
"Running 'eas update:configure' will attempt to fix this issue. Do you want us to run the command for you?"
```
Or:
```
"Running 'eas update:configure' will try to automatically add the missing configuration. Do you want us to run the command for you?"
```
Or even something else, along those lines.

Reply:
After the latest changes the content of this message has been altered, since I have changed the logic. Let's resolve that for now, and we can get back to this in the re-review, if I get confirmation, that only calling the `app.json` edit was a correct approach here.